packages/cozy-stack-client/src/SharingCollection.js

```diff
@@ -18,21 +18,45 @@ const normalizeSharing = sharing => normalizeDoc(sharing, SHARING_DOCTYPE)
  * @property {string=} update
  * @property {string=} remove
  */
+
 /**
  * @typedef {object} Recipient An io.cozy.contact
  */
+
+/**
+ * @typedef {object} Group An io.cozy.contacts.groups
+ */
+
 /**
  * @typedef {object} Sharing An io.cozy.sharings document
```

Comment:
The convention, if this is the io.cozy.sharings document is to call is IoCozySharings.

Reply:
I had a doubt, thanks for the clarification 👍